Gemfile.lock

```diff
@@ -83,30 +83,31 @@ GEM
       rake (>= 10.4, < 14.0)
     apipie-rails (0.5.18)
       rails (>= 4.1)
-    arbre (1.2.1)
-      activesupport (>= 3.0.0)
-    ast (2.4.0)
+    arbre (1.3.0)
+      activesupport (>= 3.0.0, < 6.1)
+      ruby2_keywords (>= 0.0.2, < 1.0)
+    ast (2.4.1)
     aws-eventstream (1.1.0)
-    aws-partitions (1.326.0)
-    aws-sdk-core (3.98.0)
+    aws-partitions (1.381.0)
+    aws-sdk-core (3.109.1)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.33.0)
-      aws-sdk-core (~> 3, >= 3.71.0)
+    aws-sdk-kms (1.39.0)
+      aws-sdk-core (~> 3, >= 3.109.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.67.1)
-      aws-sdk-core (~> 3, >= 3.96.1)
+    aws-sdk-s3 (1.83.0)
+      aws-sdk-core (~> 3, >= 3.109.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.1.4)
-      aws-eventstream (~> 1.0, >= 1.0.2)
+    aws-sigv4 (1.2.2)
+      aws-eventstream (~> 1, >= 1.0.2)
     axiom-types (0.1.1)
       descendants_tracker (~> 0.0.4)
       ice_nine (~> 0.11.0)
       thread_safe (~> 0.3, >= 0.3.1)
-    bcrypt (3.1.13)
+    bcrypt (3.1.16)
```

Comment:
### [bcrypt](https://github.com/codahale/bcrypt-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.1.13 → 3.1.16

[_~~change-log~~, [source-code](https://github.com/codahale/bcrypt-ruby)_]





<details>
<summary>Commits</summary>

A change of **35** commits. See the full changes on [the compare page](https://github.com/codahale/bcrypt-ruby/compare/v3.1.13...v3.1.16).

These are the first 10 commits:
- (27e1689) [start calibration from the minimum cost supported by the alg…](https://github.com/codahale/bcrypt-ruby/commit/27e168927a10d0028366fe364c1aafe1b62a25b4)
- (2875dbf) [Merge pull request #206 from sergey-alekseev/master](https://github.com/codahale/bcrypt-ruby/commit/2875dbf4359ef9ce88775f4aae67aaa011f930fa)
- (653b6fc) [fix original repos issue #201 compilation on non-i386 platfo…](https://github.com/codahale/bcrypt-ruby/commit/653b6fc67b82f425a885282ff663ccc643938f79)
- (2b40bb6) [Fix cppcheck style findings #213](https://github.com/codahale/bcrypt-ruby/commit/2b40bb6c52bb99b7e0f0e20f8b0cadfb30a9fde4)
- (4279fa0) [Merge pull request #212 from bfarago/master](https://github.com/codahale/bcrypt-ruby/commit/4279fa01f083b48de9a18c54423453059074ba77)
- (591776c) [Update Changeloge for v3.1.13](https://github.com/codahale/bcrypt-ruby/commit/591776c57d38494f1184e4a02dca8f98f1bf2934)
- (50060d8) [Reverse CHANGELOG order to fix #203](https://github.com/codahale/bcrypt-ruby/commit/50060d84608dde66066a7e7d680131a7eba2f7ed)
- (3382230) [CI: allow failures on bleeding edge Rubies](https://github.com/codahale/bcrypt-ruby/commit/3382230eaf16226dc9f7f1a050c37c0841b5d730)
- (e847f1f) [Merge pull request #216 from jeremy/ci-allows-head-failures](https://github.com/codahale/bcrypt-ruby/commit/e847f1ffff527905d12e43533a63abe0677a6470)
- (f48c58f) [Fix rare strdup segfault](https://github.com/codahale/bcrypt-ruby/commit/f48c58fc5e8419dd7322f6873e0faff468865c65)


</details>